### ISSUE TITLE
Add CI using pytest & Github Actions

### ISSUE THREAD

**Issue by mscho527** [oimeitei/quemb#23]
We would eventually want the example code in `example` (and possibly a few more scripts) exist as a `pytest` routine that gets checked everytime a commit (or PR) is submitted. This issue should track the necessary steps until we have a fully working CI environment set up in the repository.

**Comment by oimeitei** [comment]
Yes, I have been planning to add this. We can set up a separate `test` with testing for different things like say energy, density, ... Can you start adding some test sets (maybe those from example cases)?
